docs/file-reference.md

```diff
@@ -54,6 +56,13 @@ Each service has name and list of the containers.
 
 Name of the service.
 
+#### replicas
+| type  | required |
+|-------|----------|
+| int32 |    no    |
```

Comment:
wouldn't **u**int32 be more descriptive since any negative value is invalid. Or it would be good to mention it somewhere else